app/uk/gov/hmrc/customs/declaration/services/CommunicationService.scala

```diff
@@ -69,7 +68,6 @@ class CommunicationService @Inject()(logger: DeclarationsLogger,
       case Some(fieldsId) => Future.successful(FieldsId(fieldsId))
       case _ =>
         val msg = "No value found for clientId."
```

Comment:
An error should be logged here. also the message should be no value found for client subscription id